README.md

```diff
@@ -8,6 +8,7 @@
 </p>
 
 # shell-command
+[![Build Status](https://travis-ci.org/sp4ceb4r/shell-command.svg?branch=v0.2.0)](https://travis-ci.org/sp4ceb4r/shell-command)
```

Comment:
Remove this. Build status is displayed (L3).

If you want to change the status url to show the current branch, feel free.